app/jobs/send_confirmed_match_sms_job.rb

```diff
@@ -0,0 +1,28 @@
+class SendConfirmedMatchSmsJob < ApplicationJob
+  # TODO: Define retry_on policy: https://edgeapi.rubyonrails.org/classes/ActiveJob/Exceptions/ClassMethods.html#method-i-retry_on
+  queue_as :mailers
+
+  def perform(match_id)
+    match = Match.find(match_id)
+
+    return unless match.sms_confirmed_notification_needed?
+
+    provider = Flipper.enabled?(:sendinblue, match) ? Sms::SendInBlue : Sms::Twilio
+
+    sms_provider_id = provider.new(Sms::ConfirmedMessage.new(match)).send
```

Comment:
`send` is a [reserved keyword](https://apidock.com/ruby/Object/send) in ruby. While an explicit definition overrides it, it would still be cautious to rename this to something like `send_message`. What do you think?

Reply:
Sure!